lib/pavilion/plugins/commands/status.py

```diff
@@ -321,3 +328,90 @@ def display_history(self, pav_cfg, args):
                                                                id_final))
 
         return ret_val
+
+
+    def print_summary(self, statuses):
+        """Print_summary takes in a list of test statuses.
+        It summarizes basic state output and displays
+        the data to the user through draw_table.
+        :param statuses: state list of current jobs
+        :param outfile:
+        :rtype: int
+        """
+        # Populating table dynamically requires dict
+
+        summary_dict = {}
+        passes = 0
+        ret_val = 0
+        total_tests = len(statuses)
+        rows = []
+        fields = ['State', 'Amount', 'Percent']
+        fails = 0
+
+        # Shrink statues dict to singular keys with total
+        # amount of key as the value
+        for test in statuses:
+            if test['state'] not in summary_dict.keys():
+                summary_dict[test['state']] = 1
+            else:
+                summary_dict[test['state']] += 1
+
+            # Gathers info on passed tests from completed tests.
+            if 'COMPLETE' in test['state'] and 'PASS' in test['note']:
+                passes += 1
+
+        if 'COMPLETE' in summary_dict.keys():
+            fails = summary_dict['COMPLETE'] - passes
+            fields = ['State', 'Amount', 'Percent', 'PASSED', 'FAILED']
+
+        for key, value in summary_dict.items():
+            #  Build the rows for drawtables.
+
+            #  Determine Color.
+            if key in {'ERROR', 'FAILED', 'TIMEOUT'}:
```

Comment:
There are a bunch of states that contain 'ERROR', 'FAILED' and 'TIMEOUT'. All of them should be RED.